### ISSUE TITLE
Loopback Multipart form data

### ISSUE THREAD

**Issue by sujeshthekkepatt** [strongloop/loopback#4224]
Is there any update on handling multipart form data? My third party API send data in the form of multipart/form-data. I used Multer inside a custom middleware function but it gives '100 continue' response. And my remote method is not calling. Is there any proper solution? Or Any update on implementing support for multipart/form-data?

**Comment by emonddr** [comment]
@sujeshthekkepatt 

Thank you for reaching out to us.

Can you please provide a repo with a small example, and step by step description of the issue you encountered?

Thank you

:)

**Comment by sujeshthekkepatt** [comment]
@emonddr 
Thank you for the response.

I tried reproducing it but I couldn't. But the issue persists on the production app. This happens when an enctype header presents on the request. 
```
app.post('/example/endpoint/', multer().none(),(req,res,next)=>{
console.log('this middleware fucntions runs');
next(); //doesnt passing to my remote-endpoint
})
```

And after a few seconds, I am getting a 'request aborted' in the console. But this works fine when I created a sample. Don't know what may be causing it? Any help appreciated
